ports/mimxrt10xx/boards/feather_mimxrt1062/board.c

```diff
@@ -27,24 +27,8 @@
 
 #include "boards/board.h"
 #include "mpconfigboard.h"
-#include "fsl_iomuxc.h"
 
 void board_init(void) {
-    IOMUXC_SetPinMux(IOMUXC_GPIO_SD_B1_06_FLEXSPIA_SS0_B, 1U);
```

Comment:
When did this code go in the new version? Do we assume it's taken care by the bootloader? What if there is no bootloader?

Reply:
It's taken care of by the boot ROM that is built into the chip. Without it, this code wouldn't load from the flash.